providers/providers.go

```diff
@@ -359,10 +359,21 @@ func LatestVersion(name string) (string, error) {
 	}
 	client.Timeout = time.Duration(5 * time.Second)
 
-	res, err := client.Get("https://releases.mondoo.com/providers/latest.json")
-	if err != nil {
-		return "", err
+	maxRetries := 3
+	var res *http.Response
+
+	for attempt := 1; attempt <= maxRetries; attempt++ {
+		res, err = client.Get("https://releases.mondoo.com/providers/latest.json")
```

Comment:
This might not work as expected. Depending on the response, you must take care of the `res.Body`, e.g., : https://github.com/hashicorp/go-retryablehttp/blob/main/client.go#L771

The other option would be, to use the `retryablehttp`, but we need to make sure the HTTP proxy settings from the original  HTTP client are used.

Reply:
I would prefer if we use retryablehttp. It is very stable and we use it already in different parts of the code base

Reply:
Marking as a draft for now, will try that out when I have more time, currently occupied with fixes for exceptions